edx_proctoring/templates/proctoring/seq_timed_exam_expired.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+<div class="critical-time sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You did not complete the exam in the allotted time
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since you did not submit your exam before the time allotted expired, your work cannot be
+      considered as an exam that is eligible for credit. Any work you've completed may still be counted for
```

Comment:
Does "Any work you've completed..." refer to work within the exam, or work done elsewhere in the course?